ipc/bootstrap.sh

```diff
@@ -78,10 +89,11 @@ while (($#)); do
   case "$1" in
   --env)
```

Comment:
Was there an intention to allow for both `--env` and `--network`? If so, I guess gotta do this:

```suggestion
  --env)
  ;;
  --network)
```

Reply:
> Was there an intention to allow for both `--env` and `--network`? 

Yep, for compatibility with docs and everything. `network` fits better as this is what we have in fcli, right?